test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,53 @@
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+local module = require('testbase')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+require('user_modules/AppTypes')
+
+--- Preconditions
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+function Test:StartSDL_with_one_activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      print("Hmi initialized")
+      self:initHMI_onReady():Do(function ()
+        print("Hmi is ready")
+        self:connectMobile():Do(function ()
+          print("Mobile Connected")
+          self:startSession():Do(function ()
+            commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+--- Tests
+--- Description:
+---  HMI sends OnExitAllApplications with reason IGNITION_OFF:
```

Comment:
@LuxoftAKutsan, 
According to requirements: 
Req1. "[Data Resumption]:OnExitAllApplications(SUSPEND) in terms of resumption" 
SDL store all applications resumption-related data on the file system
Req2: "[Data Resumption]:OnExitAllApplications(IGNITION_OFF) in terms of resumption"
SDL must clean up any resumption-related data obtained after OnExitAllApplications( SUSPEND)

In case to have correct and expected SDL behavior needed for data resumption, SDL should receive SUSPEND before IGN_OFF. 
The state is also described in VC of Req2